_data/data-sponsor.yml

```diff
@@ -151,101 +151,72 @@ sponsors:
-          website: https://www.captaindental.com/
-          description: Captain Dental由Doris Yao, D.D.S 于2017年创立，我们创立的初衷就是要解决看牙贵，看牙痛，看牙水太深的问题。因此Captain Dental不仅注重高质量的治疗结果，也将顾客的满意与舒适度放在第一位。收费合理，透明也是我们一直以来贯彻的理念。成立至今我们在Google，Yelp上都是5星评分。
-          phone: 408-826-4676
-          email: frontdesk@captaindental.com
-          other: Locations in Santa Clara and San Jose
 
     - level-name: platinum
       level-heading: 白金赞助
       elements-div-style-class: col-6 col-lg-3 mb-1
       sponsor-elements:
-        - name: Steve Chen
-          image: https://res.cloudinary.com/zaigezaigu/image/upload/t_auto_eco_compression/c_scale,q_80,w_200/v1638483572/zgzg-io-website/Steve_Chen_%E7%9B%B4%E6%92%AD%E5%8F%8A%E5%BE%AE%E4%BF%A1%E7%BE%A4logo_kbn5in.jpg
-          description: Steve 作为资深房屋贷款顾问及销售经理，服务在湾区享有极好口碑。他的专长包括 Relocation Loan，涉及 RSU 收入及海外资产的贷款。最近十年 Steve 连续荣获富国银行房贷最高荣誉 President’s Club Member 头衔
-          phone: 408-805-1888
-          email: Steve.T.Chen @wellsfargo.com
-          website-name: Steve Chen Website
-          website: https://www.wfhm.com/loans/steve-chen
-        - name: LingoAce
-          image: https://res.cloudinary.com/zaigezaigu/image/upload/t_auto_eco_compression/v1639628465/zgzg-io-website/162466266_151532423508863_1416856759243389483_n_tc1a6b.jpg
-          description: LingoAce成立于2017年，是一家科技型教育公司，专注于为3-15岁海外青少儿提供沉浸式、趣味化且卓有成效的中文学习体验。课程服务现已覆盖全球100多个国家和地区，平台聚集4000+优质中文教师，赢得全世界家长的信赖和认可！<br> 2021年，LingoAce完成B轮和C轮融资，总计1.6亿美元，领跑全球在线语言教育赛道！目前正在积极扩充全球团队，希望在美国、欧洲、澳洲、东南亚地区吸纳更多优秀人才。美国硅谷热招岗位：产品经理、市场营销、用户增长等，欢迎自荐或他荐！
-          website-name: 在线中文试听礼包
-          website: https://ad.chinesecourseforkids.com/#/teen/home?type=2&channel=MA-BDG-BC-OT-ZGZG
+        - name: Anna
+          image: https://res.cloudinary.com/zaigezaigu/image/upload/t_auto_eco_compression/v1670308954/zgzg-io-website/%E8%B5%9E%E5%8A%A9/Ivy_Zhang_and_Anna_Duan_Realty_Team_s3t45u.png
+          description: Ivy Zhang & Anna Duan 团队是Coldwell Banker全职资深Top地产经纪团队，是Top Agent Network的成员，交易过上百套房屋买卖，得到客户们Zillow 和Google 全五星好评，团队资源丰富，能在房屋买卖，贷款，投资，装修，出租，设计等方面提供给客户们全方位的服务和咨询。永远将客户的利益放在首位，根据客户的具体情况，需求和预算，帮助客人制定专属地产投资规划。
+          website:
+        - name: EDS
+          image: https://res.cloudinary.com/zaigezaigu/image/upload/t_auto_eco_compression/v1670310255/zgzg-io-website/%E8%B5%9E%E5%8A%A9/EDS_Logo_black_mnrrkn.png
+          description: Enjoy Dance Studio 成立于2016年，是湾区最大的华人街舞工作室。我们的地点遍布南湾，半岛，东湾等地区。EDS为学生们提供高水平的KPOP，Hiphop，爵士，现代舞，机械舞，House，Locking，高跟鞋爵士等各类课程。特别于今年，我们开始扩展Hiphop宝宝班课程的投入和设计，受到的家长们的广泛好评。
+        - name: Lusso
+          image: https://res.cloudinary.com/zaigezaigu/image/upload/v1670309781/zgzg-io-website/%E8%B5%9E%E5%8A%A9/lusso_logo_daqkws.jpg
+          description: Lusso成立于2017年，由一批有情怀，具有创新精神的建筑、设计行业专业人士所创建。Lusso坐落于矽谷的中心，拥有12000平方尺的展厅。 Lusso品牌专注于高端全定制橱柜设计，高端欧橡地板和现代欧式瓷砖。
+        - name: Yami
+          image: https://res.cloudinary.com/zaigezaigu/image/upload/t_auto_eco_compression/v1670480747/zgzg-io-website/%E8%B5%9E%E5%8A%A9/yami1_s7icau.jpg
```

Comment:
Why is this a JPG? Use a PNG.

https://cdn.yamibuy.net/mkpl/d665113a07ce78d72928b0b67a254482_0x0.png

Upload to Cloudinary and replace with this image.